src/WingetCreateCLI/Commands/UpdateCommand.cs

```diff
@@ -192,27 +197,39 @@ public async Task<Manifests> DeserializeExistingManifestsAndUpdate(List<string>
         /// <returns>Boolean representing whether the manifest was updated successfully or not.</returns>
         public async Task<bool> ExecuteManifestUpdate(List<string> latestManifestContent, CommandExecutedEvent commandEvent)
         {
-            Manifests manifests = await this.DeserializeExistingManifestsAndUpdate(latestManifestContent);
-            if (manifests == null)
+            Manifests originalManifests = new Manifests();
+            Serialization.DeserializeManifestContents(latestManifestContent, originalManifests);
+
+            DisplayManifestPreview(originalManifests);
+
+            Manifests updatedManifests = await this.UpdateExistingManifests(originalManifests);
+            if (updatedManifests == null)
             {
                 return false;
             }
 
-            DisplayManifestPreview(manifests);
+            DisplayManifestPreview(updatedManifests);
 
             if (string.IsNullOrEmpty(this.OutputDir))
             {
                 this.OutputDir = Directory.GetCurrentDirectory();
             }
 
-            string manifestDirectoryPath = SaveManifestDirToLocalPath(manifests, this.OutputDir);
+            string manifestDirectoryPath = SaveManifestDirToLocalPath(updatedManifests, this.OutputDir);
 
             if (ValidateManifest(manifestDirectoryPath))
             {
                 if (this.SubmitToGitHub)
                 {
+                    if (!this.CheckForUpdatedInstallerHash(originalManifests, updatedManifests))

```

Comment:
>CheckForUpdatedInstallerHash [](http://example.com/codeflow?start=30&length=28)

I believe this will always succeed, since both have references to the same installers I think. Either way, you should add a UT for this

Reply:
Thanks for catching my mistake. I reverted the code back to using DeserializeExistingManifestsAndUpdate which has its own copy of the manifest object. I believe that this case should be covered by that UpdateAndVerifyUpdatedProperties() unit test. 
That unit test is comparing an original manifest with a new updated one and verifying the change. I believe that this should be enough to satisfy this case.

Reply:
But that's a positive-only test, we should have a negative test too which ensures that if no hashes have changed after the update, then we fail

Reply:
Good idea, I have added a negative test to make sure we check for this. This requires additional multifile test manifests just fyi

Reply:
Looks like this check won't apply/will fail if the source manifest is a singleton. Might make sense to factor out the singleton->multi manifest stuff so you can call it for the old and new manifests

Reply:
I decided to add an additional check in the verifyinstallerhash method to look for the installer hash in the singleton manifest. That way we can still use our normal singleton test cases for the unit tests. lmk what you think

Reply:
Nice, that works well